### ISSUE TITLE
Convert the features (in bits vectors) into a different representation

### ISSUE THREAD

**Issue by andresilvapimentel** [deepchem/deepchem#3059]
I am using RDkit and DeepChem to understand the features importances of some datasets.

I splitted the dataset and featurized it using ECFP (in DeepChem) so that the features are stored in train_dataset.X object, which has the shape (1631, 1024). So, it has 1024 features as bits vectors.

Then, I created a model using Random Forest and generate the features importances.  Finally, I plotted the features against relative importance. However, as the features are bits vectors, the look of y-axis is weird using matplotlib library.

Is there any way to convert these features (in bits vectors) into a different representation (smiles of fragments, per example)? How can I code this transformation?
